values.yaml

```diff
@@ -266,7 +266,7 @@ web:
 
   image:
     repository: temporalio/ui
-    tag: latest
+    tag: 2.9.0
```

Comment:
why this change? it is hard to keep this in sync, and we will have to upgrade this repo whenever ui upgrade.

Reply:
users would need to update this similar to like grafana and prometheus image tags.
this is just so that "latest" is not used and have certain pods be able to pull a newer image tag by mistake